src/edu/washington/escience/myria/accessmethod/SQLiteAccessMethod.java

```diff
@@ -455,6 +455,11 @@ public void createIndexIfNotExists(final RelationKey relationKey, final Schema s
       throws DbException {
     throw new UnsupportedOperationException("create index if not exists is not supported in sqlite yet, implement me");
   }
+
+  @Override
+  public void dropTableIfExistsCascade(final RelationKey relationDelete) throws DbException {
```

Comment:
Please move this method adjacent to (immediately below) `dropTableIfExists` and rename `relationDelete` to `relationKey` per above.